WORKLIST.txt

```diff
@@ -6,6 +6,7 @@ DATA / GAMEPLAY:
 - implement "flavored" attacks, e.g. a _BREATHE type should read m_dwFlags to figure out which type of breath (fire, ice, &c)s
 - Implement "peaceful" monsters, (Cannot be targeted, if hit they take no damage, )
 - add town level with shops and a house to store things in (some shopkeepers may raise prices depending on your character class)
+- add an intro that explains the background / hero
```

Comment:
This idea has history
https://nethackwiki.com/wiki/Introduction
-- It's not hard to do. There is even a "TOMBSTONE" mode that brings up "modal dialogs" -- just need to throw the game into (a new) "INTRO" state with appropriate IntroDT